rpc_jobs/rpc_designate.yml

```diff
@@ -41,10 +41,17 @@
       - xenial:
           FLAVOR: "performance2-15"
           IMAGE: "Ubuntu 16.04.2 LTS prepared for RPC deployment"
+      - xenial_snapshot:
```

Comment:
The commit does not reference these changes.

Reply:
Will rewrite commit msg!